_osx_setup.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
```

Comment:
i feel like the underscore should precede all of these osx_* files or none of them